net-misc/bird/bird-2.0.10.ebuild

```diff
@@ -36,8 +36,15 @@ FILECAPS=(
 	CAP_NET_RAW				usr/sbin/bird
 )
 
+PATCHES=()
+
 src_prepare() {
+	if use elibc_musl; then
```

Comment:
Please apply it unconditionally - it should be safe.

See also https://devmanual.gentoo.org/ebuild-writing/misc-files/patches/index.html#conditional-patching.

Reply:
I didn’t know, thanks for the link! I modified the two files.